test/functional/rpc_gettransaction.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
```

Comment:
Maybe add the test in test/functional/wallet_basic.p instead?

Reply:
Ok.